src/parser/MaintainSentences.h

```diff
@@ -42,62 +42,22 @@ class ColumnSpecification final {
         return Status::Error();
     }
 
-    StatusOr<int64_t> getIntValue(Getters& getter) {
-        auto r = defaultExpr_->eval(getter);
-        if (!r.ok()) {
-            return std::move(r).status();
-        }
-        auto v = std::move(r).value();
-        if (!Value::isInt(v)) {
-            return Status::Error("Wrong type");
-        }
-        return Value::toInt(v);
-    }
-
-    StatusOr<bool> getBoolValue(Getters& getter) {
-        auto r = defaultExpr_->eval(getter);
-        if (!r.ok()) {
-            return std::move(r).status();
-        }
-        auto v = std::move(r).value();
-        if (!Value::isBool(v)) {
-            return Status::Error("Wrong type");
+    void setContext(ExpressionContext* ctx) {
+        if (defaultExpr_ != nullptr) {
+            defaultExpr_->setContext(ctx);
         }
-        return Value::toBool(v);
     }
 
-    StatusOr<double> getDoubleValue(Getters& getter) {
-        auto r = defaultExpr_->eval(getter);
-        if (!r.ok()) {
-            return std::move(r).status();
-        }
-        auto v = std::move(r).value();
-        if (!Value::isDouble(v)) {
-            return Status::Error("Wrong type");
-        }
-        return Value::toDouble(v);
+    bool hasDefault() {
+        return defaultExpr_ != nullptr;
     }
 
-    StatusOr<std::string> getStringValue(Getters& getter) {
+    OptVariantType getDefault(Getters& getter) {
         auto r = defaultExpr_->eval(getter);
```

Comment:
Using folly :: option instead of  `hasDefault` and` getDefault`  is better ？

Reply:
Here need to return the error message.